packages/scene-graph-mediator/client/src/modules/fillText.ts

```diff
@@ -0,0 +1,51 @@
+export { };
+
+declare global {
+  interface String {
+    fillRight(word: string, count: number): string;
```

Comment:
Where are these methods used ?

Reply:
These methods not use now.
I used by console debug.
As pointed out elsewhere, I will try to use another npm package